common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -476,11 +476,21 @@ trait FeatureSwitches {
   )
 
   // Owner: Simple & Coherent
-  val SupportFrontendActive = Switch(
+  val UkSupportFrontendActive = Switch(
     SwitchGroup.Feature,
     "uk-supporter-traffic-to-new-support-frontend",
     "When ON, all UK membership/contribute/support links send traffic to support.theguardian.com",
     owners = Seq(Owner.withGithub("justinpinner")),
+    safeState = On,
+    sellByDate = never,
```

Comment:
These could/should have a sell by date - even if we make it 6 months from now they aren't intended to be permanent.

Reply:
Fair point. I didn't want there to be any chance of it shutting off without our say so though. I guess a long expiry would do, but we'll be pulling these out again in the next few weeks I hope.